content/index.html

```diff
@@ -3,7 +3,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-	<meta name="generator" content="Hugo 0.121.1">
+	<meta name="generator" content="Hugo 0.108.0">
```

Comment:
Why are you changing the hugo version
https://github.com/apache/hadoop-site/blob/asf-site/content/index.html#L6

Reply:
It would be better to update the Hugo version in a separate PR, but personally I'm okay with that.

Reply:
before 6c88b68c2c7225e606bfc10cca6dee32b6a5bf6b, the Hugo version is 0.111.3. It seems my local hugo version is not match to repo, I would like to update and re-submit.